CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,45 @@
+# grammY code of conduct
+
+## Who We Want to Be
+
+We as a community want to be inclusive for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, language, personal appearance, religion, or sexual identity and orientation.
+
+We are humans.
+We meet each other with respect and dignity.
+
+If it is not obvious what this means, here are a few examples of behavior that we do not tolerate.
+
+- Trolling, insulting or derogatory comments, and personal or political attacks
+- Public or private harassment
+- Publishing others' private information, such as a physical or email
+  address, without their explicit permission
+
+## Enforcement
+
+Admins enforce our standards of acceptable behavior.
+They take appropriate and fair corrective action in response to any behavior that they deem inappropriate, threatening, offensive, or harmful.
+When they take action, they will communicate this appropriately.
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the admins at <https://t.me/grammyjs>.
+In the Telegram group chat, you can tag all available admins using the `/report` command.
```

Comment:
```suggestion
In the chat, you can tag all available admins using the `/report` command.
```

Reply:
The ` should be kept and it should be stated which chat as GitHub issues could also be seen as a way to chat about a certain topic.

Reply:
We can just call it chat and add a link to it, that makes it clear

Reply:
```suggestion
In the [chat](https://t.me/grammyjs), you can also notify the available administrators using the `/report` command.
```